includes/Services/RemoteWikiFactory.php

```diff
@@ -369,6 +372,28 @@ public function unMarkExperimental(): void {
 		$this->newRows['wiki_experimental'] = 0;
 	}
 
+	public function getExtraFieldData( string $field ): mixed {
+		return $this->extra[$field] ?? null;
+	}
+
+	public function setExtraFieldData( string $field, mixed $value ): void {
+		if ( $value !== $this->getExtraFieldData( $field ) ) {
+			$extra = $this->extra;
+			$extra[$field] = $value;
+
+			$newExtra = json_encode( $extra );
+
+			if ( $newExtra === false ) {
+				// Can not set invalid JSON data to wiki_extra.
+				return;
+			}
+
+			$this->extra = $extra;
+			$this->trackChange( $field, $this->getExtraFieldData( $field ), $value );
+			$this->newRows['wiki_extra'] = $newExtra;
```

Comment:
**Personal Note**: This definitely needs tested with more than one field at once (calling this function more than once) as unlike RequestWiki we can't have just a set all field data at once function here, so like the first version of WikiRequestManager this may have an issue with that and will need to be tested and handled some other way if so.